crates/load-test/src/main.rs

```diff
@@ -10,262 +10,155 @@
 //! cargo run --release -p load-test -- -H http://127.0.0.1:9545 --report-file /tmp/report.html -u 30 -r 5 -t 60 --no-gzip
 //! ```
 use goose::prelude::*;
-use rand::{Rng, SeedableRng};
-use stark_hash::StarkHash;
 
 mod requests;
+mod tasks;
 mod types;
 
-//
-// Tasks
-//
+fn register_v01(attack: GooseAttack) -> GooseAttack {
```

Comment:
I don't know if this is fairly easy to do but would it make sense to pass some enum|flags to `register_v0x` to essentially include:
- block_explorer,
- db-based methods,
- db-less methods,
- python-based methods,
- everything (block_explorer, db, !db, call&est_fee)
Ideally this would be possible at the command line level.

Reply:
You can filter by scenarios when executing the load-test by passing the `--scenarios` command line argument. `--scenarios-list` prints a list of all scenarios to select from.

Reply:
TIL: --scenarios